packages/bee-q/src/global/styles/utils/_utility.scss

```
@@ -37,3 +37,37 @@
 .bq-notification-portal bq-notification {
   @apply m-4;
 }
+
+/**
+ * Use these classes for the toast portal container
+ */
+
+.bq-toast-portal {
+  @apply fixed z-50 flex max-h-full flex-col gap-s;
+  // NOTE: on mobile we display toast full screen top / bottom
+  @apply left-0 w-full px-s sm:left-auto sm:w-fit sm:px-0;
```

I will recommend padding-m (`16px`) for mobile

```suggestion
  @apply left-0 w-full px-m sm:left-auto sm:w-fit sm:px-0;
```